### DISCUSSION TITLE
Why was the name rocket chosen?

### DISCUSSION THREAD

**Discussion by Snapstromegon** [modernweb-dev/rocket#449] [Q&A] [answered]
[Rocket](https://rocket.rs/) is already a fairly popular web framework for building server side applications in Rust. The first public release of Rocket was back in december 2016 (nearly exactly 4 years before the first commit to this repo was done) and nowadays it has nearly 2.3 million downloads in crates.io.
It's also the first Google result for "rocket web", which leads me to believe that this name was chosen knowingly and despite the potential problems coming with two identically named projects competing in a related space.

This is not about asking you to change any naming, but it might be helpful to provide some reasoning why this name was chosen and why the potential name collision was deemed acceptable.

**Comment by daKmoR** [comment, marked as answer]
yes indeed... 

we mostly choose the name as we got a hold on the `@rocket` npm namespace (that was before npm stopped refusing to transfer inactive scopes)

we did a quick search focused on the npm ecosystem and the rust framework didn't pop up back then 🙈

The first time I heard about it was when I "launched" `@rocket` via Twitter and some people pointed me to it - but by that time we already had many alpha versions published and multiple sites built with it... so we kept the name...
